### PR TITLE
openstackclient 5.5.0, build with octaviaclient

### PR DESCRIPTION
**Pull request by laserscout**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Following the addition of the heatclient package to enable `openstack stack` commands to run; in this MR I took the liberty and added another necessary package, octaviaclient which enables `openstack loadbalancer` commands [docs](https://docs.openstack.org/python-octaviaclient/victoria/cli/index.html).

But... is there possibly a better way to install these "CLI extensions" as OpenStack calls them via homebrew?

My current hack is to run `/usr/local/Cellar/openstackclient/x.y.z/libexec/bin/pip install python-${PROJECT}client` after each  upgrade, where in this case `PROJECT=octavia`.

Initially I thought of providing these extensions as an `option`. That works nicely, but it's not allowed in `homebrew-core`. Then I thought that maybe @carlocab is pointing to the right direction [here](https://github.com/Homebrew/homebrew-core/pull/73265#issuecomment-803276047), but I couldn't come up with something. Also, reading the [Python-for-Formula-Authors documentation](https://github.com/mourner/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md) didn't bring me any bright ideas.

PS: Since changes in the formula were needed, I added its livecheck and updated its resources.

### CODE REVIEW

File: Formula/openstackclient.rb [hunk 1]
```diff
@@ -6,7 +6,11 @@ class Openstackclient < Formula
   url "https://files.pythonhosted.org/packages/02/0d/18025e48934e59768a085cb4393bebd385be2c4da1d39b4095ab48a321bf/python-openstackclient-5.4.0.tar.gz"
   sha256 "21d228d600421ad5145f31178cdc8578df7d2e569f43381c654caae5a1993b67"
   license "Apache-2.0"
-  revision 3
+  revision 4
+
+  livecheck do
+    url :stable
+  end
```
Comment (carlocab) [inline Formula/openstackclient.rb:13]: I don't think a `livecheck` block is needed at all for PyPI URLs, actually:
```
❯ brew livecheck --debug openstackclient
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/openstackclient.rb

Formula:          openstackclient
Livecheckable?:   No

URL:              https://files.pythonhosted.org/packages/02/0d/18025e48934e59768a085cb4393bebd385be2c4da1d39b4095ab48a321bf/python-openstackclient-5.4.0.tar.gz
Strategy:         Pypi
URL (strategy):   https://pypi.org/project/python-openstackclient/#files
Regex (strategy): /href=.*?\/packages.*?\/python\-openstackclient[._-]v?(\d+(?:\.\d+)*(?:[._-]post\d+)?)\.t/i

Matched Versions:
5.5.0
openstackclient : 5.4.0 ==> 5.5.0
```
Speaking of which, there's a new version. Would you like to updated it to that so we can drop the revisions?

**Comment by laserscout** [comment]
LOL!!! `python-openstackclient 5.5.0` was released a few hours ago!! I'll update it and remove the livecheck block. Didn't know it's automatically added.

**Comment by carlocab** [comment]
Did you check that the resources don't need further updating for 5.5.0, btw?

**Comment by laserscout** [comment]
Yup, I've gotten in the habit of removing all resources from the formula file, allowing `brew upgrade-python-resources` to re populate them and then evaluating the git diff.

For openstackclient-5.5.0, there was no change in dependent package versions (from this zulu morning's update).

**Review by carlocab — APPROVED** [review]
Thanks, @laserscout.

**Pull request closed without merging**